modules/tensor_mechanics/test/tests/ad_anisotropic_plasticity/ad_aniso_plasticity_x.i

```diff
@@ -0,0 +1,281 @@
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 3
+
+    # Original verification nx = 10, ny = 2, nz = 2
+    nx = 10
+    ny = 2
+    nz = 2
+    xmin = 0.0
+    ymin = 0.0
+    zmin = 0.0
+
+    xmax = 10.0
+    ymax = 1.0
+    zmax = 1.0
+  []
+  [./corner_node]
+    type = ExtraNodesetGenerator
+    new_boundary = '100'
+    nodes = '3 69'
+    input = gen
+  [../]
+  [./corner_node_2]
+    type = ExtraNodesetGenerator
+    new_boundary = '101'
+    nodes = '4 47'
+    input = corner_node
+  [../]
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+  volumetric_locking_correction = true
+  order = FIRST
+  family = LAGRANGE
+[]
+
+[AuxVariables]
+  [hydrostatic_stress]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [plasticity_strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [plasticity_strain_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [plasticity_strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [elastic_strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+[Variables]
+  [disp_x]
+  []
+  [disp_y]
+  []
+  [disp_z]
+  []
+[]
+[AuxKernels]
+  [hydrostatic_stress]
+    type = ADRankTwoScalarAux
+    variable = hydrostatic_stress
+    rank_two_tensor = stress
+    scalar_type = Hydrostatic
+  []
+  [plasticity_strain_xx]
+    type = ADRankTwoAux
+    rank_two_tensor = trial_plasticity_plasticity_strain
+    variable = plasticity_strain_xx
+    index_i = 0
+    index_j = 0
+  []
+  [plasticity_strain_xy]
+    type = ADRankTwoAux
+    rank_two_tensor = trial_plasticity_plasticity_strain
+    variable = plasticity_strain_xy
+    index_i = 0
+    index_j = 1
+  []
+  [plasticity_strain_yy]
+    type = ADRankTwoAux
+    rank_two_tensor = trial_plasticity_plasticity_strain
+    variable = plasticity_strain_yy
+    index_i = 1
+    index_j = 1
+  []
+  [elastic_strain_xx]
+    type = ADRankTwoAux
+    rank_two_tensor = elastic_strain
+    variable = elastic_strain_xx
+    index_i = 1
+    index_j = 1
+  []
+  [sigma_xx]
+    type = ADRankTwoAux
+    rank_two_tensor = stress
+    variable = stress_xx
+    index_i = 1
+    index_j = 1
+  []
+[]
+
+[Functions]
+  [pull]
+    type = PiecewiseLinear
+    x = '0 1e1 1e8'
+    y = '0 -4e2 -4e2'
+  []
+[]
+
+[Modules/TensorMechanics/Master]
+  [all]
+    strain = FINITE
+    add_variables = true
```

Comment:
you have `add_variables = true` but also add the variables explicitly above. Check inputs and avoid doing that.